Sources/StreamChatUI/ChatMessageList/ChatMessage/Decorators/ChatMessageHeaderDecoratorView.swift

```diff
@@ -0,0 +1,51 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+public struct ChatMessageHeaderDecoratorViewContent {
+    let message: ChatMessage
+    let channel: ChatChannel
+    let dateFormatter: MessageDateSeparatorFormatter
+    let shouldShowDate: Bool
+    let shouldShowUnreadMessages: Bool
+}
```

Comment:
Don't forget to put the variables publics as well as the constructor 👍 
```suggestion
public struct ChatMessageHeaderDecoratorViewContent {
    public let message: ChatMessage
    public let channel: ChatChannel
    public let dateFormatter: MessageDateSeparatorFormatter
    public let shouldShowDate: Bool
    public let shouldShowUnreadMessages: Bool
    
    public init()...
}
```